modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -39,6 +39,54 @@ Cosign prompts you for a password, and creates a Kubernetes secret.
 
 . Store the encrypted `cosign.key` private key and the `cosign.password` decryption password in the `signing-secrets` Kubernetes secret. Ensure that the private key is stored as an encrypted PEM file of the `ENCRYPTED COSIGN PRIVATE KEY` type.
 
+
+[id="chains-signing-secrets-skopeo_{context}"]
+== Signing using skopeo
+
+You can generate keys using skopeo and use them in the `cosign` signing scheme with {tekton-chains}:
+
+. Install link:https://github.com/containers/skopeo[skopeo].
+
+. Generate a public/private key pair.
++
+[source,terminal]
+----
+$ skopeo generate-sigstore-key --output-prefix <mykey>
+----
++
+Replace `<mykey>` with a key name of your choice. Skopeo prompts you for a passphrase for the private key. It then creates the key files named `<mykey>.private` and `<mykey>.pub`.
+. Encode the key files using base64.
++
+[source,terminal]
+----
+$ base64 <mykey>.pub > b64.pub
```

Comment:
IMHO we should use `base64 -w 0` everywhere so that the output is wrapped on a single line

Reply:
@PuneetPunamiya is it ok to add -w 0 to base64 commands?